content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+Next, we are going to log in to Kots Admin to complete the install and deploy the application. Since this guide assumes you have a level of familiarity with Kots, we are not going to go into detail on every step to get the application deployed.
+
+![Log-In](/images/guides/kots/ha-cluster-log-in.png)
+
+Log in to the Aplication Administration Console using the address and password from the install. Once you have logged in using the provided password, you will need to provide a license. The license is generated in the Vendor Portal and this guide assumes you already know how to do this. There are no options that need to be enabled or turned on in order to support an HA cluster, so if you already have a devlopment license, it should work for this exercise.
+
+![Add-License](/images/guides/kots/ha-cluster-add-license.png)
+
+Once you upload your license it may take a few minutes to load and continue with the deployment.
+
+
+![License-Load](/images/guides/kots/ha-cluster-license-load.png =250x250)
+
+Once the license is loaded, the next window will depend on the application being deployed as this window renders whatever has been codified in the application's [Config.yaml](https://kots.io/reference/v1beta1/config/).
+
+After the configuration window, you should see the preflight checks and eventually land on a window similar to what is shown below
+
+![KotsAdmin](/images/guides/kots/ha-cluster-app-dployed.png)
+
+
+### Verify the Deployment
+
+To see everything that has been deployed, ssh into one of the nodes and run:
+
+```shell
+kubectl get pods --all-namespaces -o wide
+```
+
+
+The output will show you all the pods running on all namespaces and on which node. Since we only have a one node cluster, all pods should be running on node 1. Check the status of the pods to make sure there are no issues.
+
+![AllPods](/images/guides/kots/ha-cluster-all-pods-one-node.png)
+
+If you are following along, you should be able to access the application by browsing to http://<ip-address-of-ha-proxy>. The sample app uses flask 'routes' to call various methods to interact with Postgres. To test if it can write to the database, use the  `/sql-check`   route which will check connection to the default database and list the databases are avaialble to the 'postgres' user. To write to the database, first use the `/sql-create` route which creates a database (appdiretdb) with a table (tblrecords) in it. Finally, use the `/sql-add` route to add a row the table with the timestamp. Once the database and table are created, simply use this route to add more records to the database.
+
+To verify that the data is in fact being written to Postgres, install pgAdmin and create a connection to the database using the Load Balancer's IP address. Once conneted, you should see the database and table. To retrieve records you can run a new query like this:
```

Comment:
What do you think about making this more general? E.g. instead of requiring `pgAdmin`, why not suggest "the postgres client of your choice"? It might help as well to include a basic `psql` command for folks to connect from a command line, and show them how to get the connection details for the DB embedded in appdirect?

I am not sure this is necessary, just thinking out loud. I guess it depends how many folks end up following along w/ AppDirect vs using their own app.

Reply:
I met you half way... I added a bit about using 'Postgres client of your choice' but also noted that the screenshots are from pgAdmin.